lesson-6-object/index.js

```diff
@@ -3,23 +3,75 @@ void readmeLoader();
 
 //TASK-1
 
+let person = {
+    firstName: 'Ali',
+    lastName: 'Kenzo',
+    age: '23',
+    hello: function() {
+        console.log('Hello my name is: ' + this.firstName + ' ' + this.lastName);
+    }
+}
+console.log('TASK-1', person.hello());
+
 //TASK-2
 
+person.familyMembers = ['Veli', 'Hasan'];
+console.log('TASK-2', person);
+
 //TASK-3
 
+person.house = {
+    country: 'Nepal',
+    city: 'Kathmandu',
+    zipCode: '36950'
+}
+console.log('TASK-3', person);
+
 //TASK-4
 
+function addField(name, value) {
+    person[name] = value;
+}
+addField('weight', '60kg');
+console.log('TASK-4', person);
+
 //TASK-5
 
+function addFamilyMember(newPerson) {
+    person.familyMembers.push(newPerson);
+}
+addFamilyMember('Kerime');
+console.log('TASK-5', person);
+
 //TASK-6
 
+function addField2(name, value) {
+    person[name] = value;
+}
```

Comment:
You already have created the same function in line 32. Please use it and remove this duplicate. Fix TASK-4 and then use person object **_`person.addField`_** method to add this `eyeColor` field